index.js

```diff
@@ -15,79 +16,93 @@ exports.handler = (event, context, callback) => {
+                    commit_msg += '<table style="font-family: monospace, \'Courier New\', Courier; font-size: 12px; margin: 0;">';
 
                     for(var i = 0; i < file.lines.length; i++){
                         //Build HTML EMAIL
                         var change = file.lines[i];
 
                         if(change.add){
-                            email += '<tr style="background-color:#ddffdd;">';
+                            commit_msg += '<tr style="background-color:#ddffdd;">';
                         } else if (change.del){
-                            email += '<tr style="background-color:#ffdddd;">';
+                            commit_msg += '<tr style="background-color:#ffdddd;">';
                         } else if (change.normal){
-                            email += '<tr">';
+                            commit_msg += '<tr">';
                         }                
 
                         if(change.add || change.del || change.normal){
-                            email += '<td style="border-right:1px solid lightgray;">';
-                            email += (change.normal) ? change.ln2 : change.ln;
-                            email += '</td>';   
+                            commit_msg += '<td style="border-right:1px solid lightgray;">';
+                            commit_msg += (change.normal) ? change.ln2 : change.ln;
+                            commit_msg += '</td>';   
                             //Deal with tabs and spaces for emails
                             var content = escape(change.content);
                             content = content.replace(/\t/g,'\u00a0 \u00a0'); 
                             content = content.replace(/ /g, '\u00a0');   
 
                             if(change.add === true){
-                                email += '<td>+' + content + '</td>';
+                                commit_msg += '<td>+' + content + '</td>';
                             } else if (change.del){
-                                email += '<td>-' + content + '</td>';   
+                                commit_msg += '<td>-' + content + '</td>';   
                             } else {
-                                email += '<td>' + content + '</td>';   
+                                commit_msg += '<td>' + content + '</td>';   
                             }
-                            email += '</tr>';
+                            commit_msg += '</tr>';
                         }
                             
-                        
                     };
                                    
-                    email+='</table><br/>';
-                });
+                    commit_msg+='</table><br/>';
 
-                email+='</body></html>';
-                return Promise.resolve({email:email,commit:e});
+                };
+                //console.log('commit_msg: ' + commit_msg);
+                email += commit_msg;
+                callback();
```

Comment:
@jwu2 
If this callback() is called too early, it might prematurely terminate the lambda function. AWS doesn't guarantee immediate termination so the function might still run for a bit.